spec/lib/govuk_publishing_components/components/component_wrapper_helper_spec.rb

```diff
@@ -215,6 +215,21 @@
       expect(component_helper.all_attributes).to eql(expected)
     end
 
+    it "does not accept an invalid dir value" do
+      error = "dir attribute (false) is not recognised"
+      expect {
+        GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(dir: "false")
+      }.to raise_error(ArgumentError, error)
+    end
+
+    it "accepts valid dir attribute value" do
```

Comment:
Trying to remember how this works, so I could be wrong, but does this also need a test for `I've set an initial value for dir and I'm trying to override it`?

Reply:
Good spot, will add a test - thanks